diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/SuppressTest.kt

```diff
@@ -0,0 +1,22 @@
+package org.cqfn.diktat.ruleset.utils
+
+import org.cqfn.diktat.ruleset.rules.IdentifierNaming
+import org.cqfn.diktat.util.LintTestBase
+import org.junit.jupiter.api.Test
+
+class SuppressTest : LintTestBase(::IdentifierNaming) {
```

Comment:
Please add more tests with various locations of annotation: on file level, on some top-level declarations etc.

Reply:
Please check if it is possible to suppress warnings for the entire file using `@file:Suppress("...")`

Reply:
This is an important feature, we need more tests: that other annotations do not cause false suppressions, that `@Suppress` without parameters or with other parameters doesn't cause false suppressions, file-level annotations etc.